frontend/src/app/crm/leads/leads.component.ts

```diff
@@ -22,16 +24,25 @@ export class LeadsComponent implements OnInit {
    dataSource : MatTableDataSource<Lead>;
 
    @ViewChild(MatPaginator, {static: true}) paginator: MatPaginator;
+   @ViewChild(MatSort, {static: true}) sort: MatSort;
+
    constructor(private leadService : LeadService) { }
 
    ngOnInit() : void {
       this.getAllLeads();
    }
 
    getAllLeads(): void {
-      this.leadService.getAllLeads().subscribe((leads) => {
-        this.dataSource = new MatTableDataSource(leads);
-        this.dataSource.paginator = this.paginator;
-      });
+
+         this.leadService.getAllLeads()
+         .subscribe((leads) => {
+
+          this.dataSource = new MatTableDataSource(leads);
+          this.dataSource.paginator = this.paginator;
+          this.dataSource.sort = this.sort;
+
+   });
```

Comment:
Add indentation every time you enter a new scope/block. So for example here, I would indent once after `getAllLeads(): void {`, and then once more after `.subscribe((leads) => {` . Look here for a better [example](https://google.github.io/styleguide/jsguide.html#formatting-function-expressions).